catalog/configurable-gauge/README.md

```diff
@@ -0,0 +1,68 @@
+# Gauge mod
+
+This is a mod example demonstrating a gauge written in [`TypeScript`](https://www.typescriptlang.org/), rendered with [`d3`](https://d3js.org/) and bundled with [`esbuild`](https://esbuild.github.io/).
```

Comment:
A catalog Mod should perhaps not be regarded an "example demonstrating".
"This is a gauge mod written in..."

Reply:
Btw, same „example“  language in all the other catalog mods READMEs.